R/basedata.R

```diff
@@ -187,6 +195,21 @@ basedata = R6::R6Class("basedata",
     get_group_count = function () {
       self$call_server("bd_get_group_count")
     },
+    calcs_are_valid = function () {
+      cache = get_indices_metadata()
+      # TODO: ARG ERROR CHECKING
+    },
+    get_indices_metadata = function (cache_type) {
```

Comment:
I think cache_type can be hard coded as `calculations_metadata` inside the function given we are returning the full structure.  
Later we can pass arguments to access metadata for specific calculations or indices.